scripts/run_docker_build.sh

```diff
@@ -18,14 +18,20 @@ CONDARC
 )
 
 cat << EOF | docker run -i \
-                        -v ${REPO_ROOT}/recipes:/conda-recipes \
+	                -v ${REPO_ROOT}:/staged-recipes \
                         -a stdin -a stdout -a stderr \
                         $IMAGE_NAME \
                         bash || exit $?
 
 # Copy the host recipes folder so we don't ever muck with it
-cp -r /conda-recipes /conda-recipes-without-example
-rm -rf /conda-recipes-without-example/example
+cp -r /staged-recipes/recipes /conda-recipes
+
+# Find the recipes from master in this PR and remove them.
+echo "Finding recipes merged in master and removing them from the build."
+pushd /staged-recipes/recipes > /dev/null
+git fetch https://github.com/conda-forge/staged-recipes.git master
```

Comment:
Added these because CircleCI had a very old stale `master` branch from months ago due to its caching of the git repo. The other CIs never needed them. Turns out there is a simple strategy to workaround the caching on CircleCI. Implemented in PR ( https://github.com/conda-forge/staged-recipes/pull/1467 ). This should make this strategy more robust to changes on the remote.